source/documentation/testing-govuk-pay.md

```diff
@@ -9,6 +9,11 @@ When testing, you’ll need to ensure:
  - REST calls succeed with 200 API codes
  - you’ve tested the user journey from your service to the payments platform using end-to-end/smoke tests
 
+You can make payments using HTTP with sandbox accounts, when testing. 
```

Comment:
It’s only the service that can be HTTP; the rest (interactions with Pay via the API and web browser) has to be HTTPS